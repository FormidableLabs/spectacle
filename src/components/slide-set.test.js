import React, { Component } from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import SlideSet from "./slide-set";

const _mockContext = function () {
  return {};
};

class MockSlide extends Component {
  render() {
    return <div>Slide Content</div>;
  }
}

describe("<SlideSet />", () => {
  test("should render correctly", () => {
    const wrapper = mount((
      <SlideSet>
        <MockSlide />
        <MockSlide />
      </SlideSet>
    ), { context: _mockContext() });
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
