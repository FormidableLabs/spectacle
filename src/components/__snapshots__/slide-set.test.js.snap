// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SlideSet /> should render correctly 1`] = `
<SlideSet
  hasSlideChildren={true}
>
  <div>
    <MockSlide>
      <div>
        Slide Content
      </div>
    </MockSlide>
    <MockSlide>
      <div>
        Slide Content
      </div>
    </MockSlide>
  </div>
</SlideSet>
`;
